apps/trading/components/navbar/navbar.tsx

```diff
@@ -20,15 +20,15 @@ export const Navbar = ({ theme, toggleTheme }: NavbarProps) => {
   }));
   const tradingPath = marketId ? `/markets/${marketId}` : '/markets';
   return (
-    <div className="px-4 flex items-stretch border-b border-neutral-300 dark:border-neutral-400 bg-black">
```

Comment:
This line was deliberately lighter than the dark theme default so the header is clearly separate to the content below:
![image](https://user-images.githubusercontent.com/1325606/189880103-7d8ccfcb-fbab-495e-a2a3-af772bc73feb.png)

Reply:
Not sure about this one, for me the difference is so subtle that it doesn't really achieve that and instead just makes it look a little off by introducing a new border color